css/amp.css

```diff
@@ -144,31 +144,47 @@ html.i-amphtml-ios-embed.i-amphtml-ios-overscroll > #i-amphtml-wrapper {
   display: inline-block;
 }
 
-.i-amphtml-layout-nodisplay {
+.i-amphtml-layout-nodisplay,
+[layout=nodisplay]:not(.i-amphtml-layout-nodisplay)
```

Comment:
do you need the `:not(.i-amphtml-layout-nodisplay)` here? , i don't think it makes a difference

Reply:
> Once it is resolved, the selector will fail to apply, so everything keeps the same specificity as before.